modules/ROOT/pages/http/http-connector.adoc

```diff
@@ -3,4 +3,155 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Release Notes: xref:release-notes::connector/connector-http.adoc[HTTP Connector Release Notes]
\ No newline at end of file
+Release Notes: xref:release-notes::connector/connector-http.adoc[HTTP Connector Release Notes]
+
+The HTTP connector allows declaring HTTP servers that can listen to
+requests and trigger flows, as well as HTTP clients that can communicate with any
+HTTP service. This means that the HTTP connector effectively allows you to both
+expose and consume HTTP based APIs. 
+
+Examples of HTTP based APIs are that the HTTP connector powers both APIkit
```

Comment:
@george-hoffer, this are not examples of HTTP based APIs. These are the MuleSoft products to create APIs and to consume them.

Reply:
I was trying to apply context to the sentence describing these. Do we need to keep this sentence if they aren't directly related to the HTTP connector? ("For example, APIkit and REST Connect both use the HTTP connector internally to work with APIs.")

Reply:
My idea of this overview is to explain the raw capabilities of the connector and how it maps to what people expect when thinking about HTTP, hence all the server and client talk. Because the subject is intertwined with APIs, which is our main business, I think it's important to note that we actually have larger tooling/products on the area and the HTTP connector is just a part of it.

Reply:
We should ideally link to docs on those subjects but I have no idea where those are.